website/source/docs/configuration/storage/postgresql.html.md

```diff
@@ -42,6 +42,19 @@ CREATE TABLE vault_kv_store (
 CREATE INDEX parent_path_idx ON vault_kv_store (parent_path);
 ```
 
+Store for HAEnabled backend
+
+```sql
+CREATE TABLE vault_ha_store (
+  ha_key                                      TEXT COLLATE "C" NOT NULL,
+  ha_identity                                 TEXT COLLATE "C" NOT NULL,          
+  ha_value                                    TEXT COLLATE "C",  
+  valid_until                                 TIMESTAMP NOT NULL,
+  CONSTRAINT ha_key PRIMARY KEY (ha_key)
+);
+```
+
+
```

Comment:
Can you add a helper function for PG versions prior to 9.5 that lack upsert feature, like we do below with vault_kv_put?

Reply:
Hold off on this.  I think we're ok with mandating 9.5+ for HA.

Reply:
I thinks so. 9.5 becomes end of life 2020. We should try to keep simple. Its not worth supporting 9.5.

However possibly we should assert it and raise error on <9.5.

Reply:
There is no mentioning of the need for 9.5 in the doc, I ll address that.

Reply:
Thanks!